ProjFS.Mac/PrjFSKextTests/KextLogMock.h

```diff
@@ -0,0 +1,10 @@
+#ifndef KextLogMock_h
+#define KextLogMock_h
+
+inline void KextLog_Error(const char* fmt, ...) {}
```

Comment:
Someday we could get fancy with testing logging, but I'll leave that as a future exercise.